### ISSUE TITLE
DetailsList failing Semantics Accessibility Check

### ISSUE THREAD

**Issue by taparikh** [OfficeDev/office-ui-fabric-react#9411]
## Actual behavior:
Role="presentation" is present when accessibility test Sematics-> Data Table is performed
The details of the test are:
This procedure uses the Web Developer browser extension.

Use the Web Developer browser extension (Outline > Outline tables) to highlight tables on the page.
1. Examine the target page to identify any data tables:
2. A data table organizes content into rows and columns to show relationships.
3. A layout table is used to visually position content without implying any relationships.
4. Use the Web Developer browser extension (Information > Display ARIA Roles) to verify that none of its elements is coded with role="presentation".
5. Record your results:
6. If you find any failures, select Fail, then add them as failure instances. Otherwise, select Pass.

## Expected behavior:
No elements in the table should have role="presentation". The Fabric control in question is DetailsList.

## Documentation describing expected behavior
The details of the test are:
This procedure uses the Web Developer browser extension.

Use the Web Developer browser extension (Outline > Outline tables) to highlight tables on the page.
1. Examine the target page to identify any data tables:
2. A data table organizes content into rows and columns to show relationships.
3. A layout table is used to visually position content without implying any relationships.
4. Use the Web Developer browser extension (Information > Display ARIA Roles) to verify that none of its elements is coded with role="presentation".
5. Record your results:
6. If you find any failures, select Fail, then add them as failure instances. Otherwise, select Pass.

![role_presentation](https://user-images.githubusercontent.com/51679982/59297586-695b5e00-8c3d-11e9-81d3-0726ecd81af2.jpg)

**Comment by KevinTCoughlin** [comment]
We had a thread offline about this last week @taparikh, are you still facing issues?

We left off with guidance from the developers of the accessibility tool you are showcasing above being:

### Data tables

Semantic elements in a data table must not be coded as presentational.

### How to fix

Do not use role="presentation" on any of the semantic elements in a data table:

```
<table>
<tr>
<th>
<td>
<caption>
<col>
<colgroup>
<thead>
<tfoot>
<tbody>
```

@taparikh, can you let us know which of the above elements are flagging the rule for `role="presentation"`?

If none are then our existing usage of the role _should be_ valid.

**Comment by taparikh** [comment]
Hi Kevin:

The following elements have role="presentation"
<div class="ms-DetailsRow-fields fields-269"> 

the DetailsList component does not contain any HTML Table elements. 

I have attached a screenshot that shows the aria roles using the tools described in my original [post.](url
<img width="1560" alt="role_presentation" src="https://user-images.githubusercontent.com/51679982/60537172-c7b7b180-9cbc-11e9-99e7-9b672f4c7da4.png">
)

**Comment by KevinTCoughlin** [comment]
It is OK to use `role="presentation" as long as the role is not present on semantic elements as per the web accessibility dev's guidance shared above.

I do not see any action for us to take based solely on the presence of `role="application" on an application page.

**Comment by taparikh** [comment]
Thank you for clarifying this. We will mark this as a non-accessibility issue & share this thread with Luke Gaylor as well.
